.github/workflows/CD.yml

```
@@ -14,14 +14,9 @@
 name: C/D
 
 on:        
-  workflow_dispatch:
-    inputs:
-      version:
-        description: 'the version number to release'
-        required: true
-      sha:
-        description: 'the github sha to release'
-        required: true
+  push:
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+*"
```

```suggestion
      - "v[0-9]+.[0-9]+.[0-9]+"
```
Do we have a need for the `*` wildcard? Currently we do not publish tags other than `v1.23.0`.

---

If that's the case, then I think it's safe to remove the wildcard

---

I'm pretty sure that's the only pattern we want to match.  We will publish additional tags, but they'd have additional content at the front, not the end, and we don't want them to trigger a release.